USAGE.md

```diff
@@ -228,7 +245,7 @@ if ($file->exists()) die('Example image already exists');
 $result = uploadFromUrl('https://upload.wikimedia.org/wikipedia/en/a/a9/Example.jpg', 'Adopt Wiki example image');
 ```
 
-### History...
+#### History...
```

Comment:
How about "Viewing the history", "Browsing history", or something along those lines?

And maybe the same for the "Other stuff" subsections: "Running custom queries", "Customizing the user agent", "Handling errors".

What do you think?

Reply:
Good suggestions, adopted most of them.